README.md

```diff
@@ -0,0 +1,44 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app), using the [Redux](https://redux.js.org/) and [Redux Toolkit](https://redux-toolkit.js.org/) template.
```

Comment:
Please add a descriptive readme file

Reply:
Add linters-config for react-redux
[linters-confic](https://github.com/microverseinc/linters-config/tree/master/react-redux)